### PR TITLE
chore(updates): fix setup drawer instructions

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/steps/next.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default SliceSimulatorPage;`,
 
 const steps = [
   InstallSliceSimulator({
-    code: `npm install --save @prismicio/react @prismicio/slice-simulator-react`,
+    code: `npm install --save @prismicio/react @prismicio/slice-simulator-react @prismicio/client@^6 @prismicio/helpers`,
```
Comment (Miniplop) [inline packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/steps/next.tsx]: in the init script we don't precise the client version : 

```
${PRISMIC_REACT_PACKAGE_NAME} ${PRISMIC_CLIENT} ${SLICE_SIMULATOR_REACT} ${PRISMIC_HELPERS}
```

So what is the good version ?

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/steps/next.tsx]: latest,
I versioned this one as we ran into issues when @pavery007 was updating a project :)

Comment (Miniplop) [reply packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/steps/next.tsx]: ok so maybe we should also update the client installation in the init also right ?

Comment (MarcMcIntosh) [reply packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/steps/next.tsx]: probably not.

Adding latest is for when they already have v5 installed, so in an old project that they are updating.

**Review by Miniplop — APPROVED** [review]

**Pull request merged**